### ISSUE TITLE
<link as="fetch"> breaks subfont

### ISSUE THREAD

**Issue by m-allanson** [Munter/subfont#25]
Hi, I recently discovered subfont and am loving it. Thanks! 

I've discovered an issue on files that contain a link element like this:

```
<link rel="preload" href="/foo.json" as="fetch" crossorigin="anonymous"/>
```

the `as="fetch"` attribute causes subfont to output warnings and not optimise fonts in the page.

### Output

```
subfont index.html -o output

✔ 0.048 secs: logEvents
 ⚠ WARN: index.html: relations/HtmlResourceHint: Not a valid type for the as-method: fetch
 ✔ 0.065 secs: loadAssets
 ⚠ WARN: index.html: relations/HtmlResourceHint: Not a valid type for the as-method: fetch
 ✔ 0.003 secs: populate
 ⚠ WARN: index.html: relations/HtmlResourceHint: Not a valid type for the as-method: fetch
 ✔ 0.001 secs: populate
 ⚠ WARN: index.html: relations/HtmlResourceHint: Not a valid type for the as-method: fetch
 ✔ 0.001 secs: populate
 ✔ 0.307 secs: subsetFonts
 ✔ 0.006 secs: writeAssetsToDisc
Output written to file:///Users/mike/dev/assetgraph-issue-demo/output/
```

### Expected output

```
subfont index.html -o output

Guessing --root from input files: file:///Users/mike/dev/assetgraph-issue-demo/
 ✔ 0.078 secs: logEvents
 ✔ 0.100 secs: loadAssets
 ✔ 8.531 secs: populate
 ✔ 0.001 secs: populate
 ✔ 0.001 secs: populate
 ✔ 1.372 secs: subsetFonts
 ✔ 0.006 secs: writeAssetsToDisc
Output written to file:///Users/mike/dev/assetgraph-issue-demo/output/
```

### Steps to reproduce

- clone https://github.com/m-allanson/assetgraph-issue-demo
- `npm install`
- `npm run subfont`

Is this expected or is there something I should be doing differently? Thanks :)

**Comment by KyleAMathews** [comment]
This is one of the options for preload in the W3C spec https://www.w3.org/TR/preload/

**Comment by Munter** [comment]
Our html resource hint implementation was a bit behind. We must have implemented from a previous spec. I can also see `media` has been replaced with `video`, `audio` and `track`. I'll have a fix ready shortly

**Comment by m-allanson** [comment]
Thanks for the speedy fix!

**Comment by Munter** [comment]
Fixed with the release of Assetgraph 4.7.3 
try triggering a reinstall that gets you this version
